workflows/nextflow_utils/samplesheets.py

```diff
@@ -49,7 +49,7 @@ def queryset_to_samplesheet(
         for field in queryset.model._meta.get_fields():
             if not field.is_relation:
                 _column_map[field.name] = SamplesheetColumnSource(
-                    lookup_string=field.name
+                    lookup_string=[field.name]
```

Comment:
Does this need to be list-wrapped like this?

Reply:
but `lookup_string` is _str | list[str]_

Reply:
probably not, it should support string